run

```diff
@@ -666,57 +666,65 @@ build() {
       fi
   done
 
-  echo "Building Holohub"
-
-  # Application to build
+  # Application or operator to build
   if [ "$1" == "" ]; then
-    print_error "Please specify which application to build."
-    echo "You can run ./run list for a full list of applications."
+    print_error "Please specify which operator or application to build."
+    echo "You can run ./run list for a full list of options."
+    exit 1
+  fi
+  project=$(basename "$1")
```

Comment:
I'm confused here, aren't we expecting `$1` to be the name of an application here, not a path?

Is this assuming some updated syntax?

Reply:
It can be both, see commit and MR description:

> ## Changes
> ...
> allow passing a path to the app/operator dir, for improved UX by leveraging
terminal autocompletion
> ...
> ## Results
> ...
> ### Build improvements
> ...
> ```
> ./run build dds_shapes_subscriber # passing name only still works
> ```
> ...
> - Can pass a path to the app or operator, improving UX by leveraging fast terminal autocomplete
> ```
> $ ./run build applications/h264/h264_endoscopy_tool_tracking/
> ...
> ```

Reply:
Great! Could you please update the `build_desc` help description to reflect that new support?

Reply:
https://github.com/nvidia-holoscan/holohub/pull/639/files#diff-acba25512100f80b56fc3ccd14c65be55d94800cda77585c5f41a887e398f9beR546